File: Conta.java
public abstract class Conta{
    private String nConta;
    private String nomeTitular;
    private String cpfTitular;
    private double saldo = 0;

    Conta(String nConta, String nomeTitular, String cpfTitular){
        this.nConta = nConta;
        this.nomeTitular = nomeTitular;
        this.cpfTitular = cpfTitular;  
    }

    public abstract double depositar(double valorDeposito);

    public abstract double sacar(double valorSaque);

    public void setnConta(String nConta) {
        this.nConta = nConta;
    }

    public String getnConta() {
        return nConta;
    }

    public void setCpfTitular(String cpfTitular) {
        this.cpfTitular = cpfTitular;
    }

    public String getCpfTitular() {
        return cpfTitular;
    }

    public void setSaldo(double saldo) {
        this.saldo = saldo;
    }

    public double getSaldo() {
        return saldo;
    }

    public void setNomeTitular(String nomeTitular) {
        this.nomeTitular = nomeTitular;
    }

    public String getNomeTitular() {
        return nomeTitular;
    }


}

File: WinFormsApp1/classes/Conta.cs
using System;

namespace WinFormsApp1
{
    public abstract class Conta 
    {
        private String nConta;
        private String senha;
        private String nomeTitular;
        private String cpfTitular;
        private Double saldo = 0;
        private Double emprestimoTaxa = 0;
        private Double emprestimo = 0;

        public Conta(String nConta, String senha, String nomeTitular, String cpfTitular)
        {
            this.nConta = nConta;
            this.nomeTitular = nomeTitular;
            this.cpfTitular = cpfTitular;
            this.senha = senha;
        }

        public abstract Double depositar(double valorDeposito);

        public abstract Double sacar(double valorSaque);

        public String Nconta { get { return nConta; } set { nConta = value; } }
        public String NomeTitular { get { return nomeTitular; } set { nomeTitular = value; } }
        public String CpfTitular { get { return cpfTitular; } set { cpfTitular = value; } }
        public Double Saldo { get { return saldo; } set { saldo = value; } }
        public String Senha { get { return senha; } set { senha = value; } }
        public Double Emprestimo { get { return emprestimo; } set { emprestimo = value; } }
        public Double EmprestimoTaxa { get { return emprestimoTaxa; } set { emprestimoTaxa = value; } }
    }
}
